catalog/core/catalog-core-commands/src/main/resources/bin/catalog-export.bat

```diff
@@ -42,12 +42,14 @@ if "%DIR%"=="" goto USAGE
 if "%FILENAME%"=="" goto USAGE
 if "%KEYSTORE%"=="" goto USAGE
 
+echo Connecting to running DDF and exporting catalog to %FILENAME%...
 call client.bat -h %HOST% "catalog:dump --include-content=%FILENAME% %DIR% "
 
 where jarsigner >nul 2>nul
 if %ERRORLEVEL% NEQ 0 (
 	echo Unable to find jarsigner, ensure that $JAVA_HOME and $JAVA_HOME/bin are set on the path.
```

Comment:
I left out an error exit code because even if jarsigner is not on the path, the script is partially successful due to the catalog:dump execution.